target/hexagon/idef-parser/macros.inc

```diff
@@ -87,11 +87,11 @@
     (fZXTN(WIDTH, 32, fBIDIR_LSHIFTR((INREG), (OFFSET), 4_8)))
 
 #define fADDSAT64(DST, A, B)                                            \
-    __a = fCAST8u(A);                                                   \
-    __b = fCAST8u(B);                                                   \
-    __sum = __a + __b;                                                  \
-    __xor = __a ^ __b;                                                  \
-    __mask = fCAST8s(0x8000000000000000ULL);                            \
+    size8u_t __a = fCAST8u(A);                                          \
+    size8u_t __b = fCAST8u(B);                                          \
+    size8u_t __sum = __a + __b;                                         \
+    size8u_t __xor = __a ^ __b;                                         \
+    size8s_t __mask = fCAST8s(0x8000000000000000ULL);                   \
```

Comment:
Does this change come from the official (original) macro?

Reply:
Pretty much, the only difference is the type of `__mask` which is `const size8u_t` in the original as the `fCAST8s` is not present there. The macros are implemented slightly differently between ours and the original.

I guess all these could be made constant in our case.

Reply:
Let's deviate as little as possible from official implementation of the macros, unless there's a reason.
Changing stuff is a liability.
If we can handle the original one, make it identical.

Reply:
Good idea, changed to the "official" version of the macro!